node/src/node_configurator/configurator.rs

```diff
@@ -2619,9 +2619,8 @@ mod tests {
     }
 
     #[test]
-    fn configuration_handles_retrieving_max_block_count_none() {
+    fn configuration_handles_retrieving_all_possible_none_values() {
         let persistent_config = PersistentConfigurationMock::new()
-            .check_password_result(Ok(true))
```

Comment:
I don't know the understanding of why this check was removed. I think it's better with a check, maybe I'm wrong on this one, let me know if we need to amend changes.